roles/update_containers/tasks/main.yml

```diff
@@ -40,10 +40,18 @@
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
-  ansible.builtin.command:
-    cmd: >-
-      oc patch csv -n openstack-operators openstack-ansibleee-operator.v0.0.1
-      --type='json' -p='[{
-      "op":"replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/1/env/0",
-      "value": {"name": "RELATED_IMAGE_ANSIBLEEE_IMAGE_URL_DEFAULT",
-      "value": {{ cifmw_update_containers_ansibleee_image_url }}}}]'
+  block:
+    - name: Get ansibleee-operator csv name
+      ansible.builtin.shell: |
```

Comment:
Nit:
`oc get csv -n openstack-operators --selector=operators.coreos.com/openstack-ansibleee-operator.<replace with namespace>` is a more k8s way of doing this

Reply:
(meaning we could switch to `ansible.builtin.command` instead \o/)

Reply:
Thank you @cjeanner @pablintino for the reviewing the pr. EDPM team have implemented the support of updating ansibleee runner image via openstack version interface. https://github.com/openstack-k8s-operators/dataplane-operator/pull/889 and https://issues.redhat.com/browse/OSPRH-7200.

We will use openstack version interface to keep the role interface consistent.